src/shared.js

```diff
@@ -3334,6 +3334,16 @@ export const formatISODateTimeDateOnly = (dateReceived) => {
     return formattedDateTimeStamp
 }
 
+export const capsEnforcer = (elemArr) => {
+    elemArr.forEach(elem => {
+        if (document.getElementById(elem)) {
+            document.getElementById(elem).addEventListener('input', (e) => {
+                document.getElementById(elem).value = (e.target.value + '').toUpperCase();
```

Comment:
`elemId` should be better than `elem`.
`document.getElementById(elem)` is used 3 times. It can be better to query once and refer to the element afterwards.
I think element.value is string type, so `+ ''` may not be necessary for type conversion.